lib/cirro_io/client/reward.rb

```diff
@@ -0,0 +1,7 @@
+module CirroIO
+  module Client
+    class Reward < Base
+      has_one :app_worker
```

Comment:
It `belongs_to :app_worker`, actually. Therefore, it requests `/v1/app-workers/:app_worker_id/payouts` by default. Thus, we should alter the endpoint on backend again 🤦

Reply:
I've tried to further modify [cirro#360](https://github.com/test-IO/cirro/pull/360) to get the desired behavior using `JSONAPI::Resources`, but haven't succeeded yet.

Reply:
I don't understand you here. Why do you assume that it requests `/v1/app-workers/:app_worker_id/payouts` by default? I don't think so. 
Take a look at client/gig.rb it is the same there with `worker_filter` and it is working fine with `/v1/gigs`